connector_builder_mcp/_validation_testing.py

```diff
@@ -249,6 +251,10 @@ def execute_stream_test_read(
         bool,
         Field(description="Whether to include actual record data in the response"),
     ] = False,
+    include_record_stats: Annotated[
+        bool,
+        Field(description="Include basic statistics on record properties"),
+    ] = False,
```

Comment:
Default to enabled, since this doesn't hit the context window as hard.

```suggestion
    ] = True,
```